### PR TITLE
ssh-keygen: refresh page

### PR DESCRIPTION
**Pull request by marek22k**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 9 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages/common/ssh-keygen.md [hunk 1]
```diff
@@ -34,3 +34,7 @@
 - Change the type of the key format (for example from OPENSSH format to PEM), the file will be rewritten in-place:
 
 `ssh-keygen -p -N "" -m {{PEM}} -f {{~/.ssh/OpenSSH_private_key}}`
+
+- Retrieve public key from secret key:
+
+`ssh-keygen -y -f {{OpenSSH_private_key}}`
```
Comment (kbdharun) [inline pages/common/ssh-keygen.md:40]:
```suggestion
`ssh-keygen -y -f {{~/.ssh/OpenSSH_private_key}}`
```

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution and welcome to tldr. Can you sign the CLA from the above command for the PR to go through?

Also, our pages can only contain a maximum of 8 examples currently, feel free to merge 2 examples for a successful CI.

**Comment by marek22k** [comment]
I have copied the command from https://askubuntu.com/questions/53553/how-do-i-retrieve-the-public-key-from-a-ssh-private-key. Can I still sign the CLA?

**Comment by marek22k** [comment]
```
-----BEGIN PGP SIGNED MESSAGE-----
Hash: SHA512

tldr-pages contributor agreement

(1) I assert that any past, present and future contributions I make to the tldr-pages project are my own work, and that I have the right to license them.

(2) I agree to license all my contributions to the tldr-pages project under the terms of the license included in the tldr-pages repository I’m contributing to.

(3) I grant the tldr-pages organization a license to distribute my contributions under any open content license compliant with the Open Definition [a], or any open source software license approved by the Open Source Initiative [b], as applicable.

a. http://opendefinition.org/licenses

b. https://opensource.org/licenses/alphabetical
-----BEGIN PGP SIGNATURE-----

iQIzBAEBCgAdFiEEmqKBWfzrPNg7whIBfoaRRmmRCMcFAmPwsTgACgkQfoaRRmmR
CMeErQ//aQ5f2A5Y/fDajEvu8uGUGb/Mo640gktpmU4AR2r8QOB0WvOZUSkRvcRo
/GNrlV/9pfuN+hm8ekGNiDZlghj6gNAtGzvOReUuXddx3v5vmet8styMGS02f5Y9
DwL7ACegq5+1oH4nHqOD2dwm6IZ8K5++Jvg7vnw7O123QncAX1e4kx039GDwA8Jf
kzSudnhKC6yyFZ1lN4vCKSg4ps2MmMNoOEi5EMVcjheUoeAjqFkNdbzNfpVj64ZD
NuC5+po9fcDi5fFT2fbiYgio+uA5SXXQK9GHwxqGH1z6ft5UAFg6SHF4ihc+FGVx
Z0fzNk5BCHUiM/Y9KfHqXWNlEXY28CqT0WpvcfzfLS31GYuSadmb2yMunk3AmE7g
bMOI2czrTgjfVYkJs+Lhvd/4JNOyDeItucMbI89HGLBpjjqPPk371nOYjgMeS0nN
n3NfAo09PxEvLbYgSrdLe80OeITL445jTZbthrY9yGPBdRD64OjVU24iNaGr4byz
Mwy+r1Im8s2D56hnPzDuiGVs7jU9zILWAY1ZkY1zRI94DAggDpfLEnY3z0kGmIVx
G8H1QWxBKHWE9C2Wffq0agdfhXqYYFNTTiOs80TkZxTJ2YcKHKU9C9kEthjDg4tM
CVuXP/nygJh/18AvTzNYyzVJvMUgY7MMHO/EUmn1yICv4oOEr7U=
=65EO
-----END PGP SIGNATURE-----
```

**Comment by kbdharun** [comment]
> I have copied the command from https://askubuntu.com/questions/53553/how-do-i-retrieve-the-public-key-from-a-ssh-private-key. Can I still sign the CLA?

Oops, I made a typo, it is comment not command, anyways thanks for signing the CLA.

**Review by kbdharun — APPROVED** [review]

File: pages/common/ssh-keygen.md [hunk 2]
```diff
@@ -7,13 +7,9 @@
 
 `ssh-keygen`
 
-- Specify file in which to save the key:
+- Generate an ed25519 key with 100 key derivation function rounds and specify the file in which to save the key:
```
Comment (blueskyson) [inline pages/common/ssh-keygen.md:10]: I think this one is easier to read.
```suggestion
- Generate an ed25519 key with 100 key derivation function rounds and save the key to a file:
```

Comment (kbdharun) [reply pages/common/ssh-keygen.md:10]:
```suggestion
- Generate an ed25519 key with 100 key derivation function rounds and save the key to a specific file:
```
How about this?

**Comment by marek22k** [comment]
Thanks for the reminder! Should I squash the commits?

**Comment by marek22k** [comment]
No, its still need a review. Maybe someone can change its state to "review needed"?!

**Review by Magrid0 — APPROVED** [review]

File: pages/common/ssh-keygen.md [hunk 3]
```diff
@@ -7,13 +7,9 @@
 
 `ssh-keygen`
 
-- Specify file in which to save the key:
+- Generate an ed25519 key with 100 key derivation function rounds and save the key to a specific file:
```
[the remark this answers is not in the document]
Comment (Magrid0) [reply pages/common/ssh-keygen.md:10]: yes, that's actually clearer

[the remark this answers is not in the document]
Comment (SethFalco) [reply pages/common/ssh-keygen.md:10]: Actaully we can scrap this suggestion and go for one of the ones above. I missed them before, sorry!

https://github.com/tldr-pages/tldr/pull/9874#discussion_r1111217867

I think this is better.

File: pages/common/ssh-keygen.md (hunk 2 above)
Comment (SethFalco) [reply pages/common/ssh-keygen.md]: I'm more keen blueskuson's versions here.

Refer to: https://github.com/tldr-pages/tldr/pull/9874#discussion_r1162514532

File: pages/common/ssh-keygen.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (marek22k) [reply pages/common/ssh-keygen.md:10]: The two proposals differ only in the word `specific`, right? However, saving to a file could also mean that you cannot freely choose the file name. With `specific` it sounds to me more like you can choose the filename. On the other hand, you can see from the command that you are saving it to a specific file.
As I understood it, the majority is now for the proposal without `specific`, right?

**Review by pixelcmtd — APPROVED** [review]
